src/skycoin/config.go

```diff
@@ -220,7 +220,7 @@ func NewNodeConfig(mode string, node NodeParameters) NodeConfig {
 		WebInterfaceCert:  "",
 		WebInterfaceKey:   "",
 		WebInterfaceHTTPS: false,
-		EnabledAPISets:    api.EndpointsRead,
+		EnabledAPISets:    api.EndpointsDefault,
```

Comment:
`READ` & `TXN`

Reply:
@olemis `EndpointsDefault ` is `READ,TXN`.

Reply:
right ...